site/source/user-manual/service-guides/nextcloud/nextcloud-apps/nc-notes.rst

```diff
@@ -0,0 +1,49 @@
+.. _nc-notes:
+
+===============
+Nextcloud Notes
+===============
+`Nextcloud Notes <https://apps.nextcloud.com/apps/notes>`_ is software for managing and syncing your notes across devices.  First set up your :ref:`client devices<nextcloud-setup>`, so that Nextcloud will automatically keep your notes synced.
+
+Clients available for:
+
+- Mobile
+    - Android
+    - iOS
+
+- Desktop:
+    - Unix Terminal
+
+NC App Setup
+------------
+#. Begin by downloading Notes from your Nextcloud service's App Store (Top-right menu -> Apps).
+
+    .. tip:: Use the "Search" function to quickly find the App you are looking for
+
+#. Once installed/enabled, you will get a new icon in your top menu for "Notes."  Click it to get started.
+
+    - You can now use the web UI to record your notes.
+
+    .. tip:: Markdown is supported
+
+Desktop & Mobile Setups
+-----------------------
+The real beauty of Notes is being able to sync to your mobile devices for use on the go.
+
+Nextcloud Notes (Android)
+.........................
+#. Download the Nextcloud Notes app from your favorite app store (Aurora, F-Droid, or Play Store).
+
+#. On the opening screen, select your existing Nextcloud account.  If one does not exist, make sure to start with your :ref:`client setup<nextcloud-setup>` or you can hit "Add Account," then create an app passcode QR under Nextcloud's main top-right menu -> Personal Settings -> Security and scan it to grant access.
```

Comment:
```suggestion
#. On the opening screen, select your existing Nextcloud account.  If one does not exist, make sure to start with your :ref:`client setup<nextcloud-setup>` or you can hit "Add Account," then create an app passcode QR under Nextcloud's main top-right menu -> Personal Settings -> Hamburger (3 lines) Menu > Security > Devices & Sessions > _Create new app password_ and scan it to grant access.
```

Reply:
Is this on mobile?  I hadn't considered that there were 2 different operations